OlatunjiYso/NairobiJavaDev
# #156005187: handle configuration changes

#### What does this PR do?
This PR handles changes in device configuration
#### Description of Task to be completed?
- Make Developer class implement Parcelable
- Save a instance of the main activity before destroying it
- Look for a saved instance of activity before creating new one
#### How should this be manually tested?
- pull this branch
- run the app on an emulator
- simulate movements to change screen orientation
- see it maintain the previous state before changing
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
[story Id #156005187](https://www.pivotaltracker.com/n/projects/2155295)
#### Screenshots (if appropriate)
![portrait](https://user-images.githubusercontent.com/27797745/45578226-1607db00-b878-11e8-8fe9-416b95820f31.png)
![landscape](https://user-images.githubusercontent.com/27797745/45578232-20c27000-b878-11e8-9374-2f2954dd76c3.png)

#### Questions:
